src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -85,6 +85,38 @@
     [:span {} (get-in config ["methodRepoMethod" "methodVersion"])]]
    (clear-both)])
 
+(react/defc MCDeleteComp
```

Comment:
Let's call this `DeleteButton` or `DeleteMethodConfigButton` or `DeleteMethodConfig`.